build-logic/plugins/src/main/kotlin/AndroidApplicationConventionPlugin.kt

```diff
@@ -32,15 +32,25 @@ class AndroidApplicationConventionPlugin : Plugin<Project> {
 
         extensions.configure<ApplicationExtension> {
             // TODO: Handle flavors. Currently implemented in `variants.gradle.kts` script
+
             namespace = AndroidApp.id
             configureKotlinAndroid(this)
+
+            val isFDroidRelease = (project.properties["isFDroidRelease"] as? String)?.toBoolean() ?: false
+
             defaultConfig {
-                AndroidApp.setRootDir(projectDir)
+                AndroidApp.setRootDir(project.projectDir)
+
+                val versionNameBasedOnFDroid = if (isFDroidRelease) {
```

Comment:
```suggestion
                val resolvedProductVersionName = if (isFDroidRelease) {
```

```suggestion
                val resolvedVersionName = if (isFDroidRelease) {
```

I cannot think of a better name but, maybe something like those options would work for both cases, fdroid and not fdroid